master/static/srv/proto_get_trials_plus.sql

```diff
@@ -177,7 +178,8 @@ SELECT
         UNION ALL
         SELECT jsonb_each(resources) FROM checkpoints_new_view c WHERE c.trial_id = t.id
     ) r
-  ) AS total_checkpoint_size
+  ) AS total_checkpoint_size,
+  LEAST(t.restarts, max_restarts) as restarts
```

Comment:
please drop a comment explaining why LEAST is needed (I get it, we save t.restarts++ even when we don't restart the last time, and changing that doesn't help us for old trials, so just make it clear)